src/scripts/compile.ts

```diff
@@ -96,7 +96,11 @@ export const getMetaData = (content: string) => {
 	return node ? node.data.parsedValue : {};
 };
 
-export const fromMarkdown = (content: string, registeredHandlers: { [type: string]: HandlerFunction }): DNode => {
+export const processMarkdown = (
+	content: string,
+	registeredHandlers: { [type: string]: HandlerFunction },
+	compileToDNode = true
```

Comment:
I try to avoid commenting on style but I really dislike adding a flag here to avoid refactoring call sites. Having `fromMarkdown` and `toH` is a clear division of tasks into something that can be obviously be composed.

Reply:
Yeah we can split those out.